README.md

```diff
@@ -323,17 +335,20 @@ app.UseForFeature(featureName, appBuilder =>
 
 Creating a feature filter provides a way to enable features based on criteria that you define. To implement a feature filter, the `IFeatureFilter` interface must be implemented. `IFeatureFilter` has a single method named `EvaluateAsync`. When a feature specifies that it can be enabled for a feature filter, the `EvaluateAsync` method is called. If `EvaluateAsync` returns `true` it means the feature should be enabled.
 
-The following snippet demonstrates how to add a customized feature filter `MyCriteriaFilter`.
+The following snippet demonstrates how to add customized feature filters `MyFirstCriteriaFilter` and `MySecondCriteriaFilter`.
 
 ``` C#
 services.AddFeatureManagement()
-        .AddFeatureFilter<"MyCriteriaFilter">();
+        .AddFeatureFilter<MyFirstCriteriaFilter>()

```

Comment:
I don't see the benefit of adding a new filter in this code snippet.

Reply:
We mentioned "Feature filters are registered by calling `AddFeatureFilter<T>` on the `IFeatureManagementBuilder` returned from `AddFeatureManagement`."
But we do not mention that AddFeatureFilter<T> will also return `IFeatureManagementBuilder`. So I register multiple filters in the example here.

Now, I like the idea to keep the simplicity. So I decided to remove it.